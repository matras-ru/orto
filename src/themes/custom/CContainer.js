export const baseClass = 'container px-4';
export const fluidClass = 'max-w-none';

export default {
    baseClass,
    fluidClass
};
