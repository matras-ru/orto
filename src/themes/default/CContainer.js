export const base = 'container';
export const modeFluid = 'max-w-none';

export default {
    base,
    modeFluid
};
