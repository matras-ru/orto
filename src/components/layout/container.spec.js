/* eslint-env jest */
import { mount } from '@vue/test-utils';
import CContainer from './container';

const BASE_CLASS = ['container', 'px-8'];

describe('CContainer', () => {
    it('default empty CContainer is the functional component and rendered', () => {
        const wrapper = mount(CContainer);
        expect(wrapper.isFunctionalComponent).toBe(true);
        expect(wrapper.is('div')).toBe(true);
        expect(wrapper.classes()).toEqual([...BASE_CLASS]);
        expect(wrapper.classes().length).toBe(2);
        expect(wrapper.text()).toEqual('');
    });
});
