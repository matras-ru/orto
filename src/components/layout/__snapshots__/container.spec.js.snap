// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CContainer default 1`] = `<div class="container"></div>`;

exports[`CContainer fluid 1`] = `<div class="container max-w-none"></div>`;
